x/likenft/keeper/msg_server_new_class_test.go

```diff
@@ -95,6 +97,7 @@ func TestNewClassNormal(t *testing.T) {
 	require.Equal(t, iscnId.Prefix.String(), classData.IscnIdPrefix)
 	require.Equal(t, iscnLatestVersion, classData.IscnVersionAtMint)
 	require.Equal(t, burnable, classData.Config.Burnable)
+	require.Equal(t, maxSupply, classData.Config.MaxSupply)
```

Comment:
let's check the whole config object at once? so it will fail if we add fields in the future

```
require.Equal(t, types.ClassConfing{...}, classData.Config)

```